oscm-initdb/Dockerfile

```diff
@@ -7,7 +7,34 @@ ENV DB_HOST_APP=oscm-db DB_PORT_APP=5432 DB_NAME_APP=bssapp DB_USER_APP=bssappus
 ENV SQL_DUMP_GLOBALS=globals.sql SQL_DUMP_BSS=bss.sql SQL_DUMP_BSSJMS=bssjms.sql SQL_DUMP_BSSAPP=bssapp.sql
 
 ENV AUTH_MODE=INTERNAL HOST_FQDN=localhost LOG_LEVEL=INFO
+ENV LOG_FILE_PATH=../logs
```

Comment:
The corresponding setting in the template file still has a literal value; it must be set to the variable placeholder in order to work.